library/core/src/cmp.rs

```diff
@@ -250,6 +256,9 @@ pub macro PartialEq($item:item) {
 /// This property cannot be checked by the compiler, and therefore `Eq` implies
 /// [`PartialEq`], and has no extra methods.
 ///
+/// Implement `Eq` in addition to `PartialEq` if it's guaranteed that
+/// `PartialEq::eq(a, a)` always returns `true`.
```

Comment:
This contradicts the text immediately above it, by only requiring the reflexive property, not the symmetric and transitive properties.

Reply:
Symmetry and transitivity are already guaranteed by `PartialEq`. The text says "Implement … in *addition* to `PartialEq`".

Should I clarify this nonetheless?

Reply:
@JanBeh Yes, please. In the bulleted list, could you add ``` (required by `PartialEq` as well)``` to both the symmetric and transitive properties? And then, in this text, could you add:
```suggestion
/// `PartialEq::eq(a, a)` always returns `true` (reflexivity), in addition to the symmetric and transitive properties already required by `PartialEq`.
```

(modulo word wrapping)

Reply:
Done.